src/main/java/paymelah/logic/commands/StatementCommand.java

```diff
@@ -0,0 +1,30 @@
+package paymelah.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.math.BigDecimal;
+
+import paymelah.model.Model;
+import paymelah.model.person.Person;
+
+/**
+ * Displays the total amount of money the user is owed from the filtered person list to the user.
+ */
+public class StatementCommand extends Command {
+
+    public static final String COMMAND_WORD = "statement";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD;
```

Comment:
Should `MESSAGE_USAGE` contain a description of the command to be consistent with the other commands (and UG)?

Reply:
We should (presumably) have a `MESSAGE_USAGE` for each command, see #76. This should be useful when implementing a `help` command, for example.

Reply:
We can probably add this later as fixes under #76 though